flight/targets/naze32/board-info/board_hw_defs.c

```diff
@@ -0,0 +1,904 @@
+/**
+ ******************************************************************************
+ * @addtogroup TauLabsTargets Tau Labs Targets
+ * @{
+ * @addtogroup CopterControl OpenPilot naze support files
```

Comment:
Probably should create a OpenNaze32 oxygen group and use that throughout these files.